op-node/rollup/derive/system_config.go

```diff
@@ -140,6 +141,22 @@ func ProcessSystemConfigUpdateLogEvent(destSysCfg *eth.SystemConfig, ev *types.L
 		}
 		destSysCfg.GasLimit = gasLimit
```

Comment:
One thought is do we want to gate this behind an `isHolocene` flag?

Some options:
- If not is holocene, error on unknown event type
- If not is holocene, recognize the event type but do not parse it
- Always parse it, the L1 attributes tx will just not utilize the parsed data

I do not like erroring on unknown event type, it adds complexity with contract versions and op-node versions. I am supportive of the current implementation, just raising this as an idea

Reply:
Doesn't seem there's much harm in allowing it to be parsed. It just means as soon as the holocene upgrade takes place, the new values will take effect, which is actually kind of convenient? (No need to wait until after the fork to set up your desired parameters....)